packages/amplication-server/src/core/billing/billing.service.ts

```diff
@@ -35,9 +35,16 @@ export class BillingService {
     private readonly analytics: SegmentAnalyticsService,
     configService: ConfigService
   ) {
-    const stiggApiKey = configService.get(Env.BILLING_API_KEY);
-    this.stiggClient = Stigg.initialize({ apiKey: stiggApiKey });
-    this.clientHost = configService.get(Env.CLIENT_HOST);
+    const isBillingEnabled = Boolean(
+      configService.get<string>(Env.BILLING_ENABLED) === "true"
+    );
```

Comment:
how about leveraging the generic type instead of Boolean?

```suggestion
    const isBillingEnabled = configService.get<boolean>(Env.BILLING_ENABLED);
```

Reply:
Look at my previous comment above - I tried it in several ways with several different values, and it didn't work for me. would be happy to understand why